src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PedigreeAnnotation.java

```diff
@@ -26,6 +27,7 @@
     private Collection<String> founderIds;
     private GATKPath pedigreeFile = null;
     private boolean hasAddedPedigreeFounders = false;
+    private PedigreeValidationType pedigreeValidationType = PedigreeValidationType.STRICT;
```

Comment:
We can't add this to the class unless its honored consistently by all code paths. As it stands, this value is used by constructors before the setter can be called, which is inconsistent - it looks like the setter is only useful to subclasses that know how to thread the call in at exactly the right time, based on the current implementation.

If controlling the validation setting in subclasses is the motivation for these changes, it would be much preferable to just add a constructor overload that sets it, and have subclasses delegate to that.